publish/dir.props

```diff
@@ -4,8 +4,13 @@
 
   <PropertyGroup>
     <ContainerName Condition="'$(ContainerName)' == ''">dotnet</ContainerName>
+    <ChecksumContainerName Condition="'$(ChecksumContainerName)' == ''">dotnet</ChecksumContainerName>
     <BaseUrl Condition="'$(BaseUrl)' == ''">https://dotnetcli.blob.core.windows.net/</BaseUrl>
     <NuGetPushTimeoutSeconds Condition="'$(NuGetPushTimeoutSeconds)' == ''">3600</NuGetPushTimeoutSeconds>
+    <BinariesRelativePath>$(Channel)/Binaries/$(ProductVersion)</BinariesRelativePath>
+    <InstallersRelativePath>$(Channel)/Installers/$(ProductVersion)</InstallersRelativePath>
+    <ChecksumsRelativePath>$(Channel)/Checksums/$(ProductVersion)</ChecksumsRelativePath>
```

Comment:
Were we going to put the checksum layout in the new structure to begin with?  That way @mellinoe doesn't need to change it on the next checkin.